dbus/subscription.go

```diff
@@ -84,6 +85,7 @@ func (c *Conn) dispatch() {
 			case "org.freedesktop.DBus.Properties.PropertiesChanged":
 				if signal.Body[0].(string) == "org.freedesktop.systemd1.Unit" {
 					unitPath = signal.Path
+					c.sendPropertiesUpdate(unitPath, signal.Body[1].(map[string]dbus.Variant))
```

Comment:
Please add a `len(signal.Body) >= 2` check and a type-check for `signal.Body[1].(map[string]dbus.Variant)`.